node/README.md

```diff
@@ -18,7 +18,18 @@ In P2P mode, node will instantiate a peer-to-peer network. It will either connec
 
 ### Building from source
 
-Node depends on the following subprojects: 
+#### Prerequisites
+* Java Development Kit (JDK), version 8
+    - We recommend using the OpenJDK 
+    - Alternatively, the [Oracle JDK](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) is an option
+* [sbt](https://www.scala-sbt.org/download.html)
+* For crypto, the [Sodium crypto library](https://github.com/jedisct1/libsodium)
+* For Rholang
+    - [CUP](http://www2.cs.tum.edu/projects/cup/install.php) 0.11b-2014-06-11 or later. See [Rholang README](https://github.com/rchain/rchain/blob/master/rholang/README.md) for notes on installation requirements.
+     - [jflex](http://jflex.de/)
+     - Build BNFC from the following commit or later: [BNFC/bnfc@7c9e859](BNFC/bnfc@7c9e859)
```

Comment:
You don't have to use an explicit link here.  If you use `<owner>/<repo>@<sha>`, it will link automatically.

The end result is the same in effect, but not writing it twice prevents situations where someone changes one and forgets to change the other.